packages/browser/src/client/orchestrator.ts

```diff
@@ -151,10 +150,8 @@ async function createTesters(testFiles: string[]) {
   const config = getConfig()
   const container = await getContainer(config)
 
-  if (config.browser.ui) {
-    container.className = ''
+  if (config.browser.ui)
     container.textContent = ''
```

Comment:
Why did you remove this? Only the fallback text should be centered

Reply:
now it is always centered, the container and the iframe will have the same size

Reply:
I've removed also the overflow auto in the container and so centering the content doesn't matter

Reply:
the iframe **should no** be centered

Reply:
Can you please explain why you did it and not just state that you did it? I can see the code, I don't see _why_. The iframe was never meant to be centered